test/plugins/test_mbsubmit.py

```diff
@@ -52,8 +48,8 @@ def test_print_tracks_output(self):
         # Manually build the string for comparing the output.
         tracklist = (
             "Open files with Picard? "
-            "01. Tag Title 1 - Tag Artist (0:01)\n"
-            "02. Tag Title 2 - Tag Artist (0:01)"
+            "01. Tag Track 1 - Tag Artist (0:01)\n"
+            "02. Tag Track 2 - Tag Artist (0:01)"
```

Comment:
Double checking here, what's the reason for this change? We're changing what the test compares to, which means it should have been failing before if there's no change to the program code. If it wasn't, then that's an internal rule in our tests that should be marked.

Reply:
See 5643358. We had two similar ways how the import files were being prepared, one of them used **Title** in the item title, the other one used **Track**. I found that **Track** is used in more tests, so I removed the other one and updated the tests accordingly.

Reply:
Cool, thanks for explaining that.